README.md

```diff
@@ -228,6 +229,9 @@ Specifying `before_stop` will execute a command before stopping the container.
 
 The `depends` option allows expressing containers that must be started before. This affects the generation of the init.d/systemd script.
 
+The `log_driver` option is passed as the --log-driver option to Docker. As of Docker 1.6, `json-file` and `syslog` are
```

Comment:
It would be useful to add these as validation rules to the type. What you describe is true of Docker, rather than of the module at present.